apps/api/src/pkg/cache/tiered.ts

```diff
@@ -69,13 +69,13 @@ export class TieredCache<TNamespaces extends Record<string, unknown>> implements
     c: Context,
     namespace: TName,
     key: string,
-    loadFromDatabase: (key: string) => Promise<TNamespaces[TName]>,
```

Comment:
Just noticed this naming is a bit wrong, since it might also be used to cache tinybird